4.queueRange.js

```diff
@@ -35,10 +35,8 @@
 
 // Write your function here
 const range = (queue) => {
-
+    let result = Math.abs(queue.peek() - queue.dequeue())
+    return result
 }
```

Comment:
To find range we would need to find the max and the min and then the difference.